omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6.rst

```diff
@@ -0,0 +1,73 @@
+OMERO.web deployed with OMERO.server CentOS 7 and IcePy 3.6
+===========================================================
+
+Please first read :doc:`../../server-centos7-ice36`.
+
+This is an example walkthrough for installing OMERO.web with OMERO.server using NGINX.
+For convenience in this walkthrough, we will use the same configuration
+options that the ones used for the installation of the OMERO.server.
+
+Installing NGINX
+----------------
+
+**The following steps are run as root.**
+
+Install NGINX, copy the NGINX OMERO configuration file into the
+NGINX configuration directory, and disable the default configuration:
+
+.. literalinclude:: ../../walkthrough/walkthrough_centos7.sh
+    :start-after: #start-nginx
+    :end-before: #end-nginx
+
+Running OMERO.web
+-----------------
+
+**The following steps are run as the omero system user.**
+
+To start the OMERO.web client run::
+
+    OMERO.server/bin/omero web start
+
+NGINX should already be running so you should be able to log in as the OMERO
+root user by going to http://localhost/ in your web browser.
+
+In addition a `systemd.service` script is available should you wish to
```

Comment:
unit files aren't scripts

Reply:
will replace by file for both ``.service`` and ``init.d`` in that's better

Reply:
:+1: Certainly fine with me.

Reply:
actually I am going to change the layout to simplify thing a bit 
now that we only have one file